backend/src/main/java/com/eyetrade/backend/controller/ArticleController.java

```diff
@@ -0,0 +1,81 @@
+package com.eyetrade.backend.controller;
+
+import com.eyetrade.backend.model.dto.ArticleDto;
+import com.eyetrade.backend.model.resource.ArticleResource;
+import com.eyetrade.backend.security.JwtUserChecker;
+import com.eyetrade.backend.service.ArticleService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/article")
+@Api(value = "Article", tags = {"Operations related with articles"})
+public class ArticleController {
+
+    @Autowired
+    private ArticleService articleService;
+
+    @Autowired
+    private JwtUserChecker jwtUserChecker;
+
+    @ApiOperation(value = "Get all articles ordered by time", response = List.class)
```

Comment:
The response should not be list it should be ArticleResource. Instead of that, you should set container as List. The line should be: 
@ApiOperation(value = "Get all articles ordered by time", response = ArticleResource.class, responseContainer = "List")